### ISSUE TITLE
Mylist is not sorting to the top

### ISSUE THREAD

**Issue by mrolfer** [andrewbrey/netflix-tweaked#8]
I am using 

Google Chrome is up to date
Version 67.0.3396.99 (Official Build) (64-bit)

and 

Macos 10.13.5

The video pause works but MyList remains at the bottom.

I have cleared my caches and restarted Chrome but to no avail.

**Comment by andrewbrey** [comment]
Related to #6, but this is not strictly a duplicate since the behaviors so far have appeared slightly different in Chrome and Firefox. For now I will leave this open as a separate issue but will attempt to address both in the next update which I plan to work on this evening.

**Comment by andrewbrey** [comment]
As described in #6, the resolution to this required essentially a rewrite of the code. The good news is that the behavior is significantly more reliable now. Please go ahead and upgrade to 1.0.9 to see the fix in action. Closing this issue.

**Comment by mrolfer** [comment]
Andrew, THANKS
Works just right now. I also really appreciate that it also does not allow the page after page of Netflix's recommendation which are almost never anything I wanted to watch anyway.
Good work!!

**Comment by mrolfer** [comment]
I also like the notice about Tweak that shows when you hit refresh!

**Comment by andrewbrey** [comment]
@mrolfer Great, glad it's working better for you, and ya I added more visual indication everywhere that the extension is working. I also implemented your recommendation to have the icon button say that it's active (when you hover over it). Turns out Chrome won't let you omit the icon entirely, so it does now just say if it's active or not when you hover over it.

If everything is working well, it would be helpful if you left a review of the extension in the store so other users can find it too 😄
